server/test/e2e.env

```diff
@@ -0,0 +1,6 @@
+# e2e database
+E2E_DATABASE_HOST=127.0.0.1
+E2E_DATABASE_PORT=5432
+E2E_DATABASE_USER=postgres
+E2E_DATABASE_PASSWORD=your_password
+E2E_DATABASE_DB=e2e_db
```

Comment:
this one is the only one that's different from the other .env file. could we just reuse the other file and add this one to it?
i'm debating whether this needs to be in the gitignore or not...
i think it does since other people will have their personal passwords in here etc.
but could just consolidate with other .env file
and we'll do a slack PSA when this lands for people to add the new env var